content/post/data-warehouse-sla.md

```diff
@@ -0,0 +1,100 @@
+>
+>The Data team promises to **[services provided]** at **[expected performance level]**. If there are any issues you can **[problem reporting]** and we promise to resolve the problem in **[response time]**. However, we established **[monitoring processes]** so that if there is an issue, we will aim to find it first and **[issue communication]**.
+
+>Sincerely,
+>The Data team
+
+
+### Stakeholder
+The first, and most important, question to answer is who your stakeholder is. You may have many stakeholders, each with different requirements. Most of them will be slight variations (e.g., the same promise with different KPIs). However, you may also have non-business area stakeholders with very specific requirements. For example, your Security & Compliance team may have concerns about PCI, PII and GDPR compliance. For these stakeholders you will need a very different SLA, one that takes precedence over your other stakeholder SLAs.
+
+For the purposes of this article I will explore a data warehouse SLA from the perspective of a business stakeholder.
+
+### Services provided
+At a 30,000 foot level you are providing data access. However, I recommend going as specific as possible when describing the data services provided. Greater specificity helps to align expectations on which fields are included and the definition of those fields. Definitions become increasingly important when the data is used to calculate individual performance and compensation. The counter point is that you don't want to go overboard - you are not providing value to the business if you spend all your time working on a data dictionary. Find the right balance between specificity and the resources available to you.   
+
+Here are four examples of services provided with increasing specificity:
+1. Business area: "The Data team promises to provide sales data"
+2. Question: "The Data team promises to provide data to answer the following questions: What were sales yesterday?..."
+3. KPIs over key dimensions: "The Data team promises to provide sales and conversion rate over time and by channel"
+4. Detailed definitions: "The Data team promises to provide the data defined in our data dictionary, found on our intranet"
+
+
+### Expected performance
+Imagine you find a magic data genie and you ask for the most detailed sales data in the world. You start flipping through the data and everything looks AMAZING... until you realize the data is two years old. That wish could have used a detailed definition of expected performance. This is the area of the SLA where we move from "what" will be delivered to "how" it will be delivered.
+
+Here are some common aspects of expected performance to include in your SLA:
+* Data freshness: How current is the data? Is the data live, updated every hour or ready at 7am every morning?
```

Comment:
This seems super valuable. I'd say it's worthwhile to invent some language around tiers of freshness: Intraday, daily, weekly and explain when each of those gets updated